atd-vze/src/Components/DataTable.js

```diff
@@ -70,7 +70,10 @@ const DataTable = ({
                       // Handle formatting the field value
                       if (fieldConfigObject.format === "dollars") {
                         fieldValue = formatCostToDollars(fieldValue);
-                      } else if (fieldConfigObject.format === "datetime") {
+                      } else if (
+                        fieldConfigObject.format === "datetime" &&
+                        fieldValue
+                      ) {
```

Comment:
this was a tiny little bug that we missed during Rose's DataTable refactoring, because it only impacts the location details page 👍